src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests/EventHandlers/UpdateChangeOfPartyRequestEventHandlerTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using AutoFixture;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using NServiceBus;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.MessageHandlers.EventHandlers;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.CommitmentsV2.Models;
+using SFA.DAS.CommitmentsV2.Types;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.UnitTests.CommandHandlers
+{
+    public class UpdateChangeOfPartyRequestEventHandlerTests
+    {
+        public UpdateChangeOfPartyRequestEventHandlerTestsFixture _fixture;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _fixture = new UpdateChangeOfPartyRequestEventHandlerTestsFixture();
+        }
+
+        [Test]
+        public async Task When_HandlingCommand_And_IsChangeOfProviderRequest_Then_ChangeOfPartyRequestIsUpdated()
```

Comment:
Can we have test for when it is change of employer- no changes are made to the change of party table